mentors/kotlin/g.json

```diff
@@ -6,5 +6,13 @@
     "link_url": "https://GurpreetSK.com",
     "avatar_url": "https://avatars1.githubusercontent.com/u/11317262?s=460&v=4",
     "bio": "I am an Android Developer based out of New Delhi, and have been working with Kotlin for over a year. I'm particularly interested in Functional & Reactive programming paradigm and like to share my knowledge and learn from others."
+  },
+  {
+    "github_username": "geoffreywiseman",
+    "name": "Geoffrey Wiseman",
+    "link_text": "Personal Website",
+    "link_url": "http://geoffreywiseman.ca",
+    "avatar_url": "https://s.gravatar.com/avatar/64480724966d63cdc631513a835c096e?s=80",
+    "bio": "I'm a software consultant in Toronto, ON, Canada. I use Kotlin for some clients to build Spring Boot microservices to deploy in containers to AWS. Since I've been enjoying using Exercism to explore languages, I decided to contribute back for Kotlin.",
```

Comment:
Bio line doesn't need a comma at the end.

Reply:
Whoops, sorry -- one downside to editing right in GitHub, less listing built in. I'll fix.